installers/rpm-code/codes/openmc.sh

```diff
@@ -17,14 +17,14 @@ openmc_main() {
     codes_yum_dependencies eigen3-devel
     codes_dependencies common
     openmc_moab
-    version=develop
     openmc_dagmc
     openmc_openmc
 }
 
 openmc_moab() {
     local p="$PWD"
-    codes_download https://bitbucket.org/fathomteam/moab.git Version5.1.0
+    # 20230827 fixes pymoab/core.pyx:1509:48: no suitable method found
```

Comment:
Why include the date? Git blame (or vc-region-history in emacs) will tell us that. 

I've waffled on this many times but I think it might be unnecessary to include the reason for pinning too. I think it may be better to write a very detailed commit message with an abundance of information rather than a terse comment.

Reply:
> Why include the date? Git blame (or vc-region-history in emacs) will tell us that.

The date is that of the commit in the download repo. This can be hard to track down (as we found with warp). It seems useful to me since it's just a sha. 

> I've waffled on this many times but I think it might be unnecessary to include the reason for pinning too. I think it may be better to write a very detailed commit message with an abundance of information rather than a terse comment.

Git blame doesn't always work well, because text gets edited over time and moved around and the exact commit message is hard to fine. I find for pinned commits, giving some reasons is useful. Given that we maintain a lot of this little build files, it's easier to put the "footnotes" inline if it is a pinned version for a every specific issues (as opposed to a package lock). I've just been finding that we end up searching around unnecessarily when we pin a version. This has happened a few times lately. Build files are different than code. A pinned version is like another if statement, and it is better to document it. Could be documented as a function, but that's awkward for the execution environment.

Reply:
> The date is that of the commit in the download repo 

What does that mean? The date we squashed and merged to master? If so, will the date be wrong (we are going to squash and merge today 8/29)? I don't know how github implements a sqaush merge but if they do a rebase then I believe the commit date changes to the day the rebase was performed. I believe then it leaves the author date the same. But I don't know what it picks for the author date when squashing. 

I feel fairly strongly that dates are better left to git. I use git extensively when doing work in the downloads repo and adding some other date I think will lead to confusion. For example, if I move that line of code in a future change should I change the date?

Reply:
> What does that mean?

It means the date in https://bitbucket.org/fathomteam/moab.git for bfccfc78e6cb3ddc02c39be437a64696bf126d86. If that commit gets deleted or incorrectly recorded (as it did with warp somehow), we have no way of knowing when something happened. This way we know the intention.

Reply:
Oh geez, that makes sense:)